src/components/Cards/PlayedCards/PlayedCards.jsx

```diff
@@ -7,7 +7,7 @@ export class PlayedCards extends React.Component {
         console.log(this.props.cards);
         return (
             <ul id="played-cards">
-                {this.props.cards.map(card => <img id={"card-" + card.cardId} key={card.cardId} className={styles.allCards} alt={"card-" + card.cardId} src={`../cards/card (${card.cardId}).jpg`}/>)}
+                {this.props.cards.map(card => <img id={"card-" + card.cardId} key={card.cardId} className={styles.allCards} alt={"card-" + card.cardId} src={require(`../cards/card (${card.cardId}).jpg`)}/>)}
```

Comment:
Did it not work without the `require`?

Reply:
Nope

Reply:
Ah well. It's not something I've done before but I thought I'd seen someone do it without `require` before.

Reply:
It works without require only when you run the client and server separately, but I think when you build the front end if it doesn't have require then the string stays the same and it breaks cause the references are wrong

Reply:
Oh OK, that makes sense. The file structure would be different for the build. In that case, `require` will work best, since it can be used in both dev and prod.